### PR TITLE
Filter events by site name

### PR DESCRIPTION
**Pull request by chrsbell**
Adds a filter pop up menu and filtering for launch sites. Right now this is limited to launch sites but I will add some filters for launches article/video/image links.

https://user-images.githubusercontent.com/15068593/167215163-dfd6dd25-c7b6-46f6-ac7c-2fd1ae14e597.mov

### CODE REVIEW

**Review by Celinarabe — APPROVED** [review]
Nice testing shown in the video 👍

File: app/src/main/java/com/example/voyagerx/ui/fragments/landing/LandingPageFragment.kt [hunk 1]
```diff
@@ -61,23 +59,18 @@ class LandingPageFragment : Fragment() {
...
-            putStringArray(LaunchDetailBundle.imageLinks, launch.image_links?.toTypedArray())
+            putString(LaunchDetailFields.id, launch.id)
+            putString(LaunchDetailFields.missionName, launch.mission_name)
+            putString(LaunchDetailFields.launchSite, launch.launch_site_long)
+            putString(LaunchDetailFields.launchDate, launch.launch_date_utc)
+            putString(LaunchDetailFields.launchYear, launch.launch_year)
+            putString(LaunchDetailFields.details, launch.details)
+            putString(LaunchDetailFields.articleLink, launch.article_link)
+            putString(LaunchDetailFields.videoLink, launch.video_link)
+            putStringArray(LaunchDetailFields.imageLinks, launch.image_links?.toTypedArray())
```
Comment (Celinarabe) [inline app/src/main/java/com/example/voyagerx/ui/fragments/landing/LandingPageFragment.kt:73]: I think you could simplify this with `let` so you don't have to keep repeating LaunchDetailFields.

```
bundle.apply {
    LaunchDetailBundle.let {
        putString(it.id, launch.id)
        ...
}

```

**Review by Celinarabe — APPROVED** [review]
One small comment that is non-blocking. Great job with this! 😄

**Review by brownyte — APPROVED** [review]
This is sickeningly good!

File: app/src/main/java/com/example/voyagerx/ui/fragments/landing/LandingPageFragment.kt (hunk 1 above)
Comment (chrsbell) [reply app/src/main/java/com/example/voyagerx/ui/fragments/landing/LandingPageFragment.kt:73]: Good idea, I always have to remind myself that this exists.

**Pull request merged**